source/upgrade/important-upgrade-notes.rst

```diff
@@ -21,6 +21,8 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v6.4                                               | New schema migrations                                                                                                                                            |
```

Comment:
@isacikgoz @agnivade @streamer45 
1. Please help provide texts for the new schema migrations for important upgrade notes once ready.
2. Please also help review the [Administration changes](https://github.com/mattermost/docs/blob/570f3c8455b550d4bf141540bb53f360243c2ab6/source/install/self-managed-changelog.md#administration) section in the changelog draft to see if anything else needs to be added to the important upgrade notes.

Reply:
@isacikgoz - Could we provide Amy with a detailed note? Things that I'd like to see are:
- What to expect?
- How to debug if something went wrong? Talk a bit about db_migrations table here.

Reply:
I've added Ibrahim's [proposed text](https://github.com/mattermost/docs/pull/5394#pullrequestreview-869543303) (with minor wording tweaks) to the changelog and important upgrade notes.

Reply:
@agnivade @isacikgoz RC-1 for v6.4 was recently cut. Let me know if additional testing for the new migration system is needed either on community-daily or v6.4-RC1.

Reply:
I'll defer to Ibrahim, but I don't think it's necessary.